ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -237,9 +305,13 @@ main() {
   if [[ $# -lt 1 ]]; then
     usage
   fi
+  if [[ -z "$GPHOME" ]]; then
+    GPHOME="/usr/local/hawq"
```

Comment:
when no GPHOME and "/usr/local/hawq", code run here:
https://github.com/interma/interma-hawq/blob/a49f85566fd136cc7940fdff15d50c985a99d2cc/ranger-plugin/scripts/enable-ranger-plugin.sh#L111

It ask user to input, so it's ok.